mmaction/datasets/pipelines/loading.py

```diff
@@ -1151,6 +1153,18 @@ def __call__(self, results):
         results['original_shape'] = imgs[0].shape[:2]
         results['img_shape'] = imgs[0].shape[:2]
 
+        # we resize the gt_bboxes and proposals to their real scale
+        if 'gt_bboxes' in results:
+            h, w = results['img_shape']
+            scale_factor = np.array([w, h, w, h])
+            gt_bboxes = results['gt_bboxes']
+            gt_bboxes = (gt_bboxes * scale_factor).astype(np.float32)
+            results['gt_bboxes'] = gt_bboxes
+            if 'proposals' in results and results['proposals'] is not None:
+                proposals = results['proposals']
+                proposals = (proposals * scale_factor).astype(np.float32)
+                results['proposals'] = proposals
```

Comment:
What's the typical datatype for `results['proposals']`? If they are float32 already, you can use

```python
results['proposals'] *= scale_factor
```

In this case, scale_factor need not to be converted to float32

Reply:
They are float64, and needs to be converted to float32